addons/account_edi_ubl_cii/models/account_edi_common.py

```diff
@@ -319,7 +319,7 @@ def _import_invoice(self, journal, filename, tree, existing_invoice=None):
                 # Normalize the name of the file : some e-fff emitters put the full path of the file
                 # (Windows or Linux style) and/or the name of the xml instead of the pdf.
                 # Get only the filename with a pdf extension.
-                name = attachment_name.text.split('\\')[-1].split('/')[-1].split('.')[0] + '.pdf'
+                name = (attachment_name.text or 'invoice.pdf').split('\\')[-1].split('/')[-1].split('.')[0] + '.pdf'
```

Comment:
You could just do
```suggestion
                name = (attachment_name.text or 'invoice.').split('\\')[-1].split('/')[-1].split('.')[0] + '.pdf'
```
But you can leave it like that ;)